specification/network/resource-manager/Microsoft.Network/stable/2020-07-01/examples/AzureWebCategoryGet.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
The name of this file should be the same with the path in the reference. This should be `azureWebCategoryGet`.

Reply:
Fixed. thanks